rules/cross-platform/command_and_control_google_drive_malicious_file_download.toml

```diff
@@ -0,0 +1,86 @@
+[metadata]
+creation_date = "2023/06/19"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/06/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies potential malicious file download and execution from Google Drive. The rule checks for download activity from
+Google Drive URL, followed by the creation of files commonly leveraged by or for malware. This could indicate an attempt
+to run malicious scripts, executables or payloads.
+"""
+false_positives = [
+    "Approved third-party applications that use Google Drive download URLs.",
+    "Legitimate publicly shared files from Google Drive."
+]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Malicious File Downloaded from Google Drive"
+note = """## Setup
+
+If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+"""
+references = ["https://intelligence.abnormalsecurity.com/blog/google-drive-matanbuchus-malware"]
+risk_score = 73
+rule_id = "a8afdce2-0ec1-11ee-b843-f661ea17fbcd"
+severity = "high"
+tags = [
+    "Elastic",
+    "Host",
+    "Linux",
+    "Windows",
+    "macOS",
+    "Threat Detection",
+    "Command and Control",
+]
+type = "eql"
+query = '''
+sequence by host.id, user.id with maxspan=30s
+[any where (
```

Comment:
#2811 will allow you to set the category field, to avoid using `any`

Reply:
Awesome! Yea the only reason it is `any` is because I allow the `event.action` to be load which is from a library event category while keeping `process` related values such as `start` in the first sequence as well.